src/messages/formatters.ts

```diff
@@ -156,3 +156,12 @@ export const TimestampStyles = {
  * The possible values, see {@link TimestampStyles} for more information.
  */
 export type TimestampStylesString = typeof TimestampStyles[keyof typeof TimestampStyles];
+
+/**
+ * Adds a ¯\\_(ツ)\_/¯ to the end of a string
+ * @param content The original content.
+ * @returns The formatted content.
+ */
+export function shrug(content: string): `${string} ¯\\_(ツ)\_/¯` {
+	return `${content.trim()} ¯\\_(ツ)\_/¯`;
```

Comment:
```suggestion
	return `${content.trimEnd()} ¯\\_(ツ)\_/¯`;
```

Reply:
Is there any difference between trim and trimEnd though? Discord will end up trimming the start of the message anyway so the result would be the same

Reply:
As I said in the comment, this isn't like the built-in slash command in the Discord App. We can also use this function in the middle of a message like
```ts
let channel!: TextChannel;
channel.send(`Some ${bold('cool')} message ${shrug('like this')}`);
```

Reply:
ah I missed the original comment, that makes sense

Reply:
Also what about making the content optional and return only the shrug if there's no content?

Reply:
Good idea, will do that